src/test/java/redis/clients/jedis/doc/SetGetExample.java

```diff
@@ -0,0 +1,35 @@
+//EXAMPLE: set_and_get
+//HIDE_START
+package redis.clients.jedis.doc;
+
+import redis.clients.jedis.Jedis;
+//REMOVE_START
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+//REMOVE_END
+
+public class SetGetExample {
+
+    @Test
+    public void run() {
+
+        Jedis jedis = new Jedis("localhost", 6379);
```

Comment:
This test will fail as Redis server running on `localhost:6379` is password protected in our test suite.

Reply:
Would it be possible to run your default test against another Redis instance with a password, but those tests here without password protection. I initially extended your BaseTest class, but removed it then again to keep the example as clean as possible. However, let me discuss with Igor. Those are multi-language examples. So if we decide for a password, then all languages (client libraries) would need to use the same password.

Reply:
Let's keep it as simple as possible and set up a separate GH Actions workflow to run "doctests" similar to what we have in redis-py https://github.com/redis/redis-py/blob/emb-examples/.github/workflows/doctests.yml

Reply:
tagged @chayim

Reply:
For tests like this, I suggest we use TestContainers, particularly the Redis Version that Julien created and currently maintains. That way, we can specify the right version of Redis and configure it in a self-contained JUnit test. See https://github.com/redis-developer/testcontainers-redis

Reply:
@bsbodden We decided by intention against such an abstraction because the test needs to act as an example for the documentation. So a reader of doc would be more interested in the vanilla way of using Redis instead of a simplified way of testing.